west.yml

```diff
@@ -93,7 +93,7 @@ manifest:
       groups:
         - hal
     - name: hal_nxp
-      revision: afd053086e9e36a413f7cbec40694733588e1b9f
+      revision: 9ad3a5fea81924b55770569ab729d026193f48d3
       path: modules/hal/nxp
```

Comment:
This needs to point to the hal PR pull/229. The revision line should be:

revision: pull/229/head